packages/java/tests/spring/react-signals/frontend/helper/NumberSignalServiceWrapper.ts

```diff
@@ -0,0 +1,14 @@
+import { NumberSignalChannel } from '@vaadin/hilla-react-signals';
+import client_1 from 'Frontend/generated/connect-client.default.js';
+
+export class NumberSignalServiceWrapper {
```

Comment:
I would suggest converting this code from a class with static methods to exported functions, so you can import them using `*`:

```ts
import * as NumberSignalServiceWrapper from 'Frontend/helper/NumberSignalServiceWrapper.ts';
```

Reply:
> ... class with static methods to a list of functions since it's gonna be the final implementation

I'm a bit confused about the _"it's gonna be the final implementation"_, since I was picturing the generator will generate client-side services in a way that this helper service wrapper will not be needed at all, so this is temporary until we add the generator plugin for endpoints methods returning a signal instance.